COCOAPODS.md

```diff
@@ -1,150 +1,381 @@
-of an Xcode project. After that the framework built from the Kotlin/Native module can be used from
-this Xcode project. If necessary, this framework is automatically rebuilt during Xcode build process.
+1. Add a dependency on a Pod library stored locally with `pod()` to `build.gradle.kts` (`build.gradle`) of your
+ project.  
+As the third parameter, specify the path to `Podspec` of the local Pod using `project.file(..)`.  
+    > You can add local dependencies on subspecs as well.  
+    > The `cocoapods` block can include dependencies to Pods stored locally and Pods from the CocoaPods repository at
+    > the same time.
+    {:.note}
 
-## Workflow
+    <div class="sample" markdown="1" theme="idea" data-highlight-only>
+    
+    ```kotlin
+    kotlin {
+        ios()
+    
+        cocoapods {
+            summary = "CocoaPods test library"
+            homepage = "https://github.com/JetBrains/kotlin"
+            pod("pod_dependency", "1.0", project.file("../pod_dependency/pod_dependency.podspec"))
+            pod("subspec_dependency/Core", "1.0", project.file("../subspec_dependency/subspec_dependency.podspec"))
+            
+            pod("AFNetworking", "~> 4.0.0")
+            pod("SDWebImage/MapKit")
+        }
+    }
+    ```
+    
+    </div>
 
-To import a Kotlin/Native module in an existing Xcode project:
+2. Re-import the project.
 
-0. Make sure that you have CocoaPods [installed](https://guides.cocoapods.org/using/getting-started.html#installation).
-We recommend using CocoaPods 1.6.1 or later.
+If you want to use dependencies on local pods from Kotlin code, import the corresponding packages.
 
-1. Configure a Gradle project: apply the `org.jetbrains.kotlin.native.cocoapods` plugin, add
-and configure the targets, and specify the required podspec fields.
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
 
-2. Run the `podspec` task. The podspec file described above will be generated.
+```kotlin
+import cocoapods.pod_dependency.*
+import cocoapods.subspec_dependency.*
+```
 
-    > In order to avoid compatibility issues during an Xcode build, the plugin requires using
-    a [Gradle wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html). 
-    To generate the wrapper automatically during execution of the `podspec` task,
-    run it with the parameter `-Pkotlin.native.cocoapods.generate.wrapper=true`.
+</div>
 
-3. Add a reference to the generated podspec in a Podfile of the Xcode project.
+You can find a sample project [here](https://github.com/Kotlin/kotlin-with-cocoapods-sample).
+ 
+## Use a Kotlin Gradle project as a CocoaPods dependency
 
-    <div class="sample" markdown="1" theme="idea" mode="ruby">
+You can use a multiplatform project with native targets as a CocoaPods dependency (Kotlin Pod). You can include such a dependency
+  in a `Podfile` of an Xcode project  by its name and path to the project directory containing the generated `Podspec`. 
```

Comment:
in the `Podfile` of an Xcode project by its name and path to the project directory containing the generated `Podspec`. 
This dependency will be automatically built (and rebuilt) along with this project. 
Such an approach simplifies importing to Xcode by removing a need to write corresponding Gradle tasks and Xcode build steps manually.

Reply:
✅